tools/romimg/src/platform.h

```diff
@@ -6,11 +6,15 @@ int GetLocalhostName(char *buffer, unsigned int BufferSize);
 unsigned int GetSystemDate(void);
 unsigned int GetFileCreationDate(const char *path);
 int GetCurrentWorkingDirectory(char *buffer, unsigned int BufferSize);
+void upperbuff(char *temp);
 
 #if defined(_WIN32) || defined(WIN32)
 #define PATHSEP '\\'
 #else
 #define PATHSEP '/'
 #endif
+#ifndef MAX_PATH
+#define MAX_PATH 4096
```

Comment:
`PATH_MAX` is available in all the OS, so use it instead and remove this from here

Reply:
windows has the name flipped `MAX_PATH` instead of  `PATH_MAX`

Reply:
I'm using `MAX_PATH` in my fork and all the CI/CD was passing.
You are probably speaking about compiling natively, not through MinGW, in theory, we shouldn't care about these platforms...

Reply:
not sure where it is defined then... everything in google says on limits.h but it's not the case